measurements/regard/regard.py

```diff
@@ -0,0 +1,149 @@
+Example 2 (returns the maximum toxicity score):
+    >>> regard = evaluate.load("regard", module_type="measurement")
+    >>> input_texts = ["these girls are silly", "these boys are lost"]
+    >>> results = toxicity.compute(predictions=input_texts, aggregation = "maximum")
+    >>> print(results)
+    {'max_regard': {'negative': 0.6691194772720337, 'positive': 0.02632979303598404, 'neutral': 0.942646861076355, 'other': 0.22687028348445892}}
+Example 3 (returns the average toxicity score):
+    >>> regard = evaluate.load("regard", module_type="measurement")
+    >>> input_texts = ["these girls are silly", "these boys are lost"]
+    >>> results = toxicity.compute(predictions=input_texts, aggregation = "average")
+    >>> print(results)
+    {'average_regard': {'negative': 0.3448680592700839, 'positive': 0.022568698041141033, 'neutral': 0.5139247514307499, 'other': 0.11863846285268664}}
+"""
+
+
+def regard(preds, regard_classifier):
+    neg_scores = []
+    pos_scores = []
+    neut_scores = []
+    other_scores = []
+    regard_scores = {}
+    all_regard = []
+    for pred in preds:
+        regard = regard_classifier(pred)
+        all_regard.append(regard)
+        for s in regard:
+            if s["label"] == "negative":
+                neg_scores.append(s["score"])
+            elif s["label"] == "positive":
+                pos_scores.append(s["score"])
+            elif s["label"] == "neutral":
+                neut_scores.append(s["score"])
+            else:
+                other_scores.append(s["score"])
+        regard_scores["negative"] = neg_scores
+        regard_scores["positive"] = pos_scores
+        regard_scores["neutral"] = neut_scores
+        regard_scores["other"] = other_scores
+    return all_regard, regard_scores
+
+
+@evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class Regard(evaluate.Measurement):
+    def _info(self):
+        return evaluate.MeasurementInfo(
+            module_type="measurement",
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "predictions": datasets.Value("string", id="sequence"),
+                }
+            ),
+            codebase_urls=[],
+            reference_urls=[],
+        )
+
+    def _download_and_prepare(self, dl_manager):
+        regard_tokenizer = AutoTokenizer.from_pretrained("sasha/regardv3")
+        regard_model = AutoModelForSequenceClassification.from_pretrained("sasha/regardv3")
+        self.regard_classifier = pipeline(
+            "text-classification", model=regard_model, top_k=4, tokenizer=regard_tokenizer, truncation=True
+        )
+
+    def _compute(
+        self,
+        predictions,
+        aggregation="all",
+    ):
+        all, scores = regard(predictions, self.regard_classifier)
```

Comment:
Would rename `all` to something else (avoid overwriting names used for built-in Python functions)